docs/content/preview/manage/backup-restore/instant-db-cloning.md

```diff
@@ -0,0 +1,292 @@
+
+![Database clone](/images/manage/backup-restore/db-clone.png)
+
+Cloning has two main use cases:
+
+- Development and testing. Because the two databases are completely isolated, you can experiment with the cloned database, perform DDL operations, read and write data, and delete the clone without impacting the original. Developers can test their changes on an identical copy of the production database without affecting its performance.
+
+- Data recovery. To recover from data loss due to user error (for example, accidentally dropping a table) or application error (for example, updating rows with corrupted data), you can create a clone of your production database from a point in time when the database was in a good state. This allows you to perform forensic analysis, export the lost or corrupted data from the clone, and import it back to the original database.
+
+## Enable database cloning
+
+To enable database cloning in a cluster, you set the YB-Master flag `enable_db_clone` to true. Because cloning is in {{<badge/tp>}}, you must also add the `enable_db_clone` flag to the [allowed_preview_flags_csv](../../../reference/configuration/yb-master/#allowed-preview-flags-csv) list.
+
+For example, to set these flags when creating a cluster using yugabyted, use the `--master_flags` option of the [start](../../../reference/configuration/yugabyted/#start) command as follows:
+
+```sh
+./bin/yugabyted start advertise_address=127.0.0.1 \
+    --master_flags "allowed_preview_flags_csv={enable_db_clone},enable_db_clone=true" \
+    --tserver_flags "ysql_hba_conf_csv={host all all 0.0.0.0/0 trust,local all all trust}"
+```
+
+You can set the runtime flags while the yb-master process is running using the yb-ts-cli [set_flag](../../../admin/yb-ts-cli/#set-flag) command as follows:
+
+```sh
+./bin/yb-ts-cli --server-address=master_host:7100 set_flag allowed_preview_flags_csv enable_db_clone
+./bin/yb-ts-cli --server-address=127.0.0.1:7100 set_flag enable_db_clone true
+```
+
+## Clone databases
+
+### Prerequisites
+
+- [Create a snapshot schedule](../../../manage/backup-restore/point-in-time-recovery/#create-a-schedule) for the database you want to clone.
+
+    For example, creating a snapshot schedule with retention period of 7 days keeps the history of the updates for the entire 7 days. After that you can create a clone of the original database as of any point in time within the specified retention period.
```

Comment:
"keeps the history of the updates for the entire 7 days. . After that you can create a clone of the original database as of any point in time within the specified retention period." -> "allows you to create a clone of the original database to any time in the past 7 days"